arch/src/x86_64/mod.rs

```diff
@@ -131,6 +134,10 @@ pub fn configure_system(
         params.0.hdr.cmd_line_ptr = cmdline_addr.raw_value() as u32;
         params.0.hdr.cmdline_size = cmdline_size as u32;
         params.0.hdr.kernel_alignment = KERNEL_MIN_ALIGNMENT_BYTES;
+        if let Some(initramfs_config) = initramfs {
+            params.0.hdr.ramdisk_image = initramfs_config.address.raw_value() as u32;
+            params.0.hdr.ramdisk_size = initramfs_config.size as u32;
+        }
```

Comment:
right, I've actually seen that after I pushed the branch, and moved that outside of the if, but then it still didn't work with a bzImage. I'm gonna update the PR with all the suggestions, and see again.

Reply:
done, check the latest code.

unfortunately, it doesn't work for me. When loading a bzImage instead of the vmlinux.bin (of the same exact kernel) it behaves as if I never specified the initramfs.

Reply:
Please rebase your code on top of #919. This should work with that fix.

Reply:
works